machine-runner/src/design/event.ts

```diff
@@ -201,23 +195,13 @@ export namespace MachineEvent {
      * // where A and B are MachineEvent.Factory
      * // results in A | B
      */
-    export type Reduce<T extends Readonly<Factory.Any[]>> = LooseReduce<T>
-
-    // =====
-    type LooseReduceToEvent<T, UNION extends MachineEvent.Any = never> = T extends Readonly<
-      [Factory<infer Key, infer Payload>, ...infer Rest]
+    export type Reduce<
```

Comment:
Nitpick: I would change the documentation from `Turns a ...` to `Reduces a subtype to`

Reply:
Fixed https://github.com/Actyx/machines/pull/41/commits/c7cb7bf5fb28778b682cf2cca45ca5bb0b367790